### DISCUSSION TITLE
MONAI label Radiology Segmentation

### DISCUSSION THREAD

**Discussion by longchingkwok331** [Project-MONAI/MONAILabel#1669] [Q&A]
https://www.youtube.com/watch?v=3HTh2dqZqew&t=247s&pp=ygUPRGVlcEVkaXQgTU9OQUkg

Hi new user here, I have watched these 2 videos, and read about the deepEdit, deepGrow and segmentation model https://arxiv.org/pdf/2305.10655.pdf. 

I see that MONAI always mention active learning strategy, is the segmentation method inside the radiology app (method used in the youtube video) using active learning strategies? 

Moreover, is the segmentation method regarded as the non interactive approach as defined in the paper: https://ar5iv.labs.arxiv.org/html/2203.12362? 

Thank you

**Comment by diazandr3s** [comment]
Hi @longchingkwok331,

Thanks for your questions.

> I see that MONAI always mention active learning strategy, is the segmentation method inside the radiology app (method used in the youtube video) using active learning strategies?

By default, the segmentation model doesn't declare the Active Learning strategy. However, you could add a strategy following the example proposed in DeepEdit model: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/deepedit.py#L185

> Moreover, is the segmentation method regarded as the non interactive approach as defined in the paper: https://ar5iv.labs.arxiv.org/html/2203.12362?

Yes, that's correct. The Segmentation model is the non-interactive model. DeepEdit and DeepGrow are the interactive ones.

Hope this helps,
